Admin/PageAdmin.php

```diff
@@ -432,6 +449,12 @@ protected function configureFormFields(FormMapper $formMapper)
                             ),
                         )
                     )
+                ->add('targetRedirectType', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array(
+                        'choices' => array(
+                            '301' => 'Permanent (301)',
```

Comment:
Like that: https://github.com/sonata-project/SonataAdminBundle/blob/f7036764853ca0c5a6994635f114eab5434194e1/Form/Extension/ChoiceTypeExtension.php#L67